.appveyor.yml

```diff
@@ -27,17 +27,17 @@ install:
       Write-Host "Installed" -ForegroundColor Green
   - ps: |
       Write-Host "Installing GStreamer..." -ForegroundColor Cyan
-      $msiPath = "$($env:USERPROFILE)\gstreamer-1.0-x86_64-1.14.4.msi"
+      $msiPath = "$($env:USERPROFILE)\gstreamer-1.0-msvc-x86_64-1.18.0.msi"
```

Comment:
since gstreamer 1.18.1 is out and mac and android builds are already done, we should update this patch to use 1.18.1 also.